app/ide-desktop/lib/client/src/paths.ts

```diff
@@ -1,11 +1,12 @@
 /** @file File system paths used by the application. */
 
+import fss from 'node:fs'
 import * as path from 'node:path'
 
 import * as electron from 'electron'
 import electronIsDev from 'electron-is-dev'
 
-import * as paths from '../paths'
+import * as paths from '../paths.js'
```

Comment:
Why this change? I think we do not have any JS sources anymore here, right? So no extension should work.

Reply:
Not needed anymore. Actually having or not JS files doesn't make a difference, you need to import TS with `js` extension.
The extension is needed when running this through `ts-node`, `esbuild` seems to not care.